index.html

```diff
@@ -1,144 +1,129 @@
-    <section class='informative' id="introduction">
-    <h2>
-        Introduction
-    </h2>
-    <p>Web developers need the ability to assess and understand the performance characteristics of their
-    applications. While JavaScript <a href='#ECMA262'>[ECMA262]</a> provides a mechanism to measure application latency (retrieving the current
-    timestamp from the <code>Date.now()</code> method), the precision of this timestamp varies between user agents.</p>
-    <p>This document defines the <a>PerformanceMark</a> and
-	<a>PerformanceMeasure</a> interfaces, and extensions to the
-	<a href="#extensions-performance-interface"><code>Performance</code></a> interface,
-    which expose a high precision, monotonically increasing timestamp so they can better measure the performance characteristics of their applications.</p>
-
+         },{
+           key: 'Mailing list',
+           data: [{
+             value: 'public-web-perf@w3.org',
+             href: 'https://lists.w3.org/Archives/Public/public-web-perf/'
+           }]
+         },{
+           key: 'Implementation',
+           data: [{
+             value: 'Can I use User Timing?',
+             href: 'http://caniuse.com/#feat=user-timing'
+           }, {
+             value: 'Test Suite',
+             href: 'http://w3c-test.org/user-timing/'
+           }, {
+             value: 'Test Suite repository',
+             href: 'https://github.com/w3c/web-platform-tests/tree/master/user-timing'
+           }]
+         }]
+    };
+  </script>
+</head>
+<body>
+  <section id="abstract">
+    <p>This specification defines an interface to help web
+    developers measure the performance of their applications by giving them access
+    to high precision timestamps.</p>
+  </section>
+  <section id="sotd">
+    <p>User Timing Level 2 replaces the first version of [[USER-TIMING]] and includes:</p>
+    <ul>
+      <li>Support for <a>PerformanceMark</a> and <a>PerformanceMeasure</a> in Web Workers [[!WORKERS]] via integration with [[HR-TIME-2]];
+      <li>Processing clarifications for mark names that reference <code>PerformanceTiming</code> interface defined in [[NAVIGATION-TIMING]].</li>
+    </ul>
+  </section>
+  <section class="informative" id="introduction">
+    <h2>Introduction</h2>
+    <p>Web developers need the ability to assess and understand the performance characteristics of their applications. While JavaScript <a href="#ECMA262">[ECMA262]</a> provides a mechanism to measure application latency (retrieving the current timestamp from the <code>Date.now()</code> method), the precision of this timestamp varies between user agents.</p>
+    <p>This document defines the <a>PerformanceMark</a> and <a>PerformanceMeasure</a> interfaces, and extensions to the <a href="#extensions-performance-interface"><code>Performance</code></a> interface, which expose a high precision, monotonically increasing timestamp so they can better measure the performance characteristics of their applications.</p>

```

Comment:
It may be worthwhile to mention here the benefits of `mark()`/`measure()` relative to `performance.now()`. Without mentioning `performance.now()`, `Date.now()` might seem like a straw man.

Perhaps mentioning Dev Tools support or support in tools like Windows Performance Analyzer and WebPageTest may be worth putting in a non-normative section. Not sure if it's out-of-scope for this document though.

Reply:
Agreed. I opened #37 to track this so we can get this PR close quicker.